hack/install-pipeline-service.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
+MODE=${MODE:-preview} parse_flags $@
```

Comment:
Should it always force `preview`? e.g. `MODE=preview parse_flags $@`

Reply:
updated